doc/sphinx/source/recipes/recipe_cvdp.rst

```diff
@@ -0,0 +1,30 @@
+CVDP Package
+============
+
+About
+-----
+
+The NCAR Climate Variability Package (CVDP) is a stand alone ncl based application for the analysis of climate variability in models and observations, see [1].
+
+Requirements
+------------
+
++ cvdp (https://github.com/NCAR/CVDP-ncl)
++ nco (optional for creating netcdf files)
+
+Configuration
+-------------
+
+To use it within the ESMValTool set the environment variable *CVDP_ROOT* to the top level directory of the CVDP package.
+
+Also adapt the *config-developer.yml* file at the keyword: *output_file* to let the preprosessor output end in
+*YYYYMM-YYYYMM.nc*
+e.g.:
+    -  output_file: '[project]_[dataset]_[mip]_[exp]_[ensemble]_[field]_[short_name]_[start_year]-[end_year]'
+    +  output_file: '[project]_[dataset]_[mip]_[exp]_[ensemble]_[field]_[short_name]_[start_year]01-[end_year]12'
```

Comment:
Instead of doing this, just create symlinks in the cvdp_wrapper.py script.

Reply:
Good! I didn't feel well to tell people to touch the `config-developer.yml` to match the naming requirements of this package. I will create links as you said.

Reply:
There is a `create_link` function in the wrapper script that creates links to the preprocessed files in the `work_dir` directory. This function takes care of the naming requirements of the cvdp package, as suggested.